packages/fxa-admin-server/package.json

```diff
@@ -16,7 +16,7 @@
     "restart": "pm2 restart pm2.config.js",
     "delete": "pm2 delete pm2.config.js",
     "test": "yarn gen-keys && yarn test-default && yarn test-e2e ",
-    "gen-keys": "node -r esbuild-register ./scripts/gen_keys.ts;",
```

Comment:
Ah yes, the scripts folder moved recently because we needed to access them in the final build. Looks like this is collateral damage from that change.

 I'll put up a PR to fix this as it looks like there might be another affected path... If it's blocking you, you can fix this by changing it to: `"gen-keys": "node -r esbuild-register ./src/scripts/gen_keys.ts;`

Reply:
Okay thanks I'll revert the change then, since it's not related to my fix to avoid confusion.